internal/git/remote.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/anchore/chronicle/internal"
 )
 
-var remotePattern = regexp.MustCompile(`(?m)\[remote "origin"](\n.*)*url\s*=\s*(?P<url>.*)$`)
```

Comment:
Just sneaking in a fix -- this was reading a `url` from the `vulnerability-match-labels` because it was multiline and had a `.*` in the middle.

Reply:
great catch